### DISCUSSION TITLE
[Help] Cards stackable for mobile size.

### DISCUSSION THREAD

**Discussion by YauKeng** [fomantic/Fomantic-UI#1444] [Views (card, comment, item, statistic,...)] [answered]
The smaller device size for Formantic is below 768px only?
If I have a `ui stackable doubling four cards`, how can I show 2 card when the device size between 550 - 768? 
Now when I resize to 767, my cards will have full width, it's too wide for me.
I would prefer when the size between 550 - 767 will have 2 cards displayed. 
Please see my uploaded image [here](https://res.cloudinary.com/myfoodkampung/image/upload/v1588241856/MyFoodKampung/eq0bnqo7dgz6tbtqtqzf.png)
and the [jsfiddle](https://jsfiddle.net/3gc1xohL/)

Can I make if 550 - 767 show 2 cards, only when extra small(<550) device size will show 1 card?

**Comment by ko2in** [comment, marked as answer]
@YauKeng You can override in `cards.overrides` file in your theme. Here's the [one](https://github.com/fomantic/Fomantic-UI/blob/master/src/themes/default/views/card.overrides) from default theme. Add your CSS for your desire breakpoints and then build your theme via gulp.

Read the [documentation](https://fomantic-ui.com/usage/theming.html) to learn how to build your theme if you've not done any theme customization.

**Comment by YauKeng** [comment]
> @YauKeng You can override in `cards.overrides` file in your theme. Here's the [one](https://github.com/fomantic/Fomantic-UI/blob/master/src/themes/default/views/card.overrides) from default theme. Add your CSS for your desire breakpoints and then build your theme via gulp.
> 
> Read the [documentation](https://fomantic-ui.com/usage/theming.html) to learn how to build your theme if you've not done any theme customization.

Thank you very much, I've figure out how to do it.
